### ISSUE TITLE
Declare a license

### ISSUE THREAD

**Issue by BenJam** [DemocracyClub/WhoCanIVoteFor#71]
Please.

**Comment by JoeMitchell** [comment]
Do you mean this bit squirrelled away on the /about page:
https://whocanivotefor.co.uk/about/

**Comment by BenJam** [comment]
Yes, but in the repo so anyone wanting to use the code base in their own project can do so with confidence.

**Comment by andylolz** [comment]
But NB that’s the data license (rather than a code license, which I think may be absent).

**Comment by JoeMitchell** [comment]
Ah, yes, I'm with you (both). Sounds good.

**Comment by symroe** [comment]
Thanks Ben, you're right to raise this. I've been putting off thinking about it, as although important I find it too complex to think about.

Do you have a specific use case for this code base that you'd like us to take in to consideration? I would like a licence that works with all our code rather than a per project one, and I generally think that the data is more important than the software.

Can you advise on what you think would be the best one? It would be nice to have an attribution share-alike licence, and for people not to make buckets of money by copying our code (good luck with that though!). I would also like to reflect the interests of the UK voters, as per the CIC community…

**Comment by BenJam** [comment]
Apologies. I had assumed (wrongly) that the URL contained a license for the code. I don't have any need for it, I just thought it would be pertinent to ask you to add one as I've putting together a little piece of research regarding the impact of civic technology and the digital infrastructure that supports it. See https://github.com/librariesio/libraries.io/issues/1019

**Comment by symroe** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Establish licence for code reuse.](https://trello.com/c/n2PtyZas/866-establish-licence-for-code-reuse)

**Comment by chris48s** [comment]
> I would like a licence that works with all our code rather than a per project one

One thing to consider here is that some DC code is already associated with BSD 3-clause or MIT licences so if you pick one licence to apply consistently across DC projects (which is a good idea), you also need to consider if there are any issues about changing the licence of code/re-licencing. I think as far as MIT is concerned, you can re-licence under a more restrictive licence. For the projects that are already licenced under BSD 3-clause, there *may* be considerations around changing the licence/picking a 'compatible' licence, but I don't know.

**Comment by BenJam** [comment]
there *might* be a platform that you can use to uncover which licences you might depend upon and thus be limited to ;)

**Comment by symroe** [comment]
Pinging @frabcus
